content/teknologi/altinnstudio/altinn-api/_index.md

```diff
@@ -7,1012 +7,61 @@ aliases:
-```
-The body of the request is currently not used by the endpoint and should preferably be nothing more than an empty string.
-
-The operation returns an updated instance with the added `CompleteConfirmation`.
-
-
-## Events
-
-### Instance events
-
-All instance events, e.g. creation, read, save, change process state, ...
-
-```http
-GET {storagePath}/instances/347829/762011d1-d341-4c0a-8641-d8a104e83d30/events
-```
-
-Example of event data.
-
-```json
-{
-    "id":"6dff32bc-0928-4ae8-937c-b362d6941c89",
-    "instanceId": "60238/5c6b1a71-2e1f-447a-ae2f-d1807dcffbfb",
-    "eventType": "deleted",
-    "created": "2019-05-02T13:08:21.981476Z",
-    "instanceOwnerPartyId": "60238",
-    "user": {
-        "userId": 338829,
-        "authenticationLevel": 1,
-        "enduserSystemId": 2
-    },
-    "process": {
-        "started": "2019-05-01T12:45:01.3233Z",
-        "startEvent": "Start_22",
-        "currentTask": {
-            "elementId": "Task_2"
-        }
-    }
-}
-```
-
-### Application events (for application owners)[^1]
-
-> **WARNING**: This section will be redesigned
-
-Selected instance events. Created, first read, change process state. Optinally specified by application developer.
-
-Events can be queried. May be piped.
-
-```http
-GET {storagePath}/applications/org/app/events?created=gte:2019-03-30&process.currentTask=Task_2
-```
-
-## Validation
-
-There are multiple endpoints that can trigger validation and provide feedback regarding the state of an instance and its data.
-Some of them validates data already uploaded and stored in Altinn while others can validate input.
+## API user groups
 
-### Validate stored instance
+There are primarily two groups of users of the Altinn APIs. The first group consists of applications and systems used by the owners of the Apps hosted by Altinn. This group is called *Application Owners*. The second group consists of organisations and people using the Apps to communicate with the application owners. This group is called *Application Users*. 
 
-This action will validate an entire instance including the all data elements already uploaded and stored in Altinn.
-Validation will be performed based on current task in the process. Validation will ensure that all required data elements, as defined by the application metadata,
-have been uploaded and are valid.
-If the result of the validation is that nothing is missing or wrong, the action will update the instance with information indicating that the instance is valid.
-The response will be an empty array. In the case of validation issues the response will provide an array of the identified issues.
-Validation of a full instance use validation of data as described in the chapter below.
+The two groups have many similar needs, but there are also differences in what type of tasks they need to be able to perform. All new APIs will be available to both groups, but some endpoints and features will normally only be used by one of them. 
```

Comment:
```suggestion
The two groups have many similar needs, but there are some differences in what type of tasks they need to perform. All new APIs will be available to both groups, but some endpoints and features will normally only be used by one of them. 
```

Reply:
Is it true that all new APIs are available to both groups? All endpoints that are authorized based on scopes won't be available to the end users.